apps/bondy/src/bondy_utils.erl

```diff
@@ -294,13 +294,19 @@ external_session_id(undefined) ->
     undefined.
 
 %% -----------------------------------------------------------------------------
-%% @doc
+%% @doc It returns the timeout in ms.
+%% - Provided timeout if it is greater than 0
+%% - wamp_max_call_timeout if the provided timeout is equals to 0
+%% - wamp_call_timeout if no timeout is provided
 %% @end
 %% -----------------------------------------------------------------------------
 timeout(#{timeout := T}) when is_integer(T), T > 0 ->
```

Comment:
OK, great, I will check it;
@aramallo FYI this error is present in **1.0.0-beta.30** release version

Reply:
Here this will never be a float as the wamp library already coerced it to an integer. Also I would argue that AUtobhan providing a float should be regarded as a bug.

Reply:
Also the spec says that a value of 0 equals to infinity but I do not agree with that. Infinite timeouts are a terrible idea. That is what we have the max timeout value as a config.

Reply:
@alejandro-miguez i am on a mobile and maybe not reading this correctly, but are we assuming a timeout less that 1000 is in seconds? That is not right , timeout is in MS always.

Reply:
Autobahn-py docs says the timeout is in seconds.
See [CallOptions](https://autobahn.readthedocs.io/en/latest/reference/autobahn.wamp.html?highlight=calloptions#autobahn.wamp.types.CallOptions).
So xbar must expect seconds, but it is true it depends on the router.
Autobahn-py does not do anything with the timeout except checking it is either an int or a float.

Reply:
Thanks @alan-ls I would propose sticking to milliseconds